### PR TITLE
ipyrad: make noarch python and use pip install

### PR DESCRIPTION
**Pull request by eaton-lab**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by eaton-lab** [comment]
The build is passing, but linting fails. If I set build number to 0 then the build fails saying there is already a build 0; if I set build number to 1 then the lint fails saying I need to reset the build number to zero...

**Comment by epruesse** [comment]
@biocondabot lint

File: recipes/ipyrad/meta.yaml [hunk 1]
```diff
@@ -12,17 +12,18 @@ source:
...
 requirements:
   host:
-    - python {{ python }}
-    - setuptools
+    - pip
+    - python
   run:
-    - python {{ python }}
+    - python
+    - pip
```
Comment (epruesse) [inline recipes/ipyrad/meta.yaml]: Does this need `pip` at runtime?

**Comment by epruesse** [comment]
> The build is passing, but linting fails. If I set build number to 0 then the build fails saying there is already a build 0; if I set build number to 1 then the lint fails saying I need to reset the build number to zero...

Not sure what happened there. Might have been a "race condition" with the other PR getting merged.

**Comment by epruesse** [comment]
@biocondabot merge

**Comment by epruesse** [comment]
Dang.

**Pull request merged**